packages/api/src/command/webhook/fitbit.ts

```diff
@@ -120,15 +113,14 @@ export const mapData = async (
     const nutrition = await provider.getNutritionData(connectedUser, startdate);
     payload.nutrition = [nutrition];
   } else if (collectionType === FitbitCollectionTypes.sleep) {
-    const sleep = await provider.getSleepData(connectedUser, dayjs(startdate).format("YYYY-MM-DD"));
+    const sleep = await provider.getSleepData(connectedUser, dayjs(startdate).format(ISO_DATE));
     payload.sleep = [sleep];
+  } else if (collectionType === FitbitCollectionTypes.userRevokedAccess) {
+    // do nothing until issue #652 is resolved
```

Comment:
Not blocking, but do you expect we'll call `mapData` with `collectionType === userRevokedAccess`? If not, we could just filter it out on `processData` before `mapData` (with a `TODO #652`) and not have this empty conditional here.

But NABD as long as we review this on #652